### PR TITLE
gnucash: update to 5.4

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/gnucash/template [hunk 1]
```diff
@@ -17,7 +17,7 @@ license="GPL-2.0-or-later"
 homepage="http://www.gnucash.org"
 changelog="https://raw.githubusercontent.com/Gnucash/gnucash/stable/NEWS"
 distfiles="https://github.com/Gnucash/gnucash/releases/download/${version}/gnucash-${version}.tar.gz"
```
Comment (sgn) [inline srcpkgs/gnucash/template:19]:
```suggestion
distfiles="https://github.com/Gnucash/gnucash/releases/download/${version}/gnucash-${version}-1.tar.gz"
```

File: srcpkgs/gnucash/template [hunk 2]
```diff
@@ -16,8 +16,8 @@ maintainer="ManfredU <mu@usselmann.it>"
 license="GPL-2.0-or-later"
 homepage="http://www.gnucash.org"
 changelog="https://raw.githubusercontent.com/Gnucash/gnucash/stable/NEWS"
-distfiles="https://github.com/Gnucash/gnucash/releases/download/${version}/gnucash-${version}.tar.gz"
-checksum=e0d04e0fd5f03f39136e1f4d941ccd0202b64a6e92418f5382cb6a6772493529
+distfiles="https://github.com/Gnucash/gnucash/releases/download/${version}/gnucash-${version}-1.tar.gz"
+checksum=9d3eb3cd63730cf34d4c14dea6921bfe57f3936e0ca6e17ffc04c38dc1b40061
```
Comment (sgn) [inline srcpkgs/gnucash/template:20]:
```suggestion
checksum=7989c2fff67ff356e99c9a4b8d90a7759c7bde73844afb0595ee1322f4a19ced
```

**Comment by cinerea0** [comment]
@sgn Please stop. I am aware of what went wrong. The distfile link and hash changed out from under me in between when I first built version 5.4 and now. I am testing to make sure that nothing else changed.

**Comment by Piraty** [comment]
reporting the failed test on 32bit seems worth it

**Comment by cinerea0** [comment]
Bug reported: https://bugs.gnucash.org/show_bug.cgi?id=799114.

**Pull request merged**